addons/onboarding/models/onboarding_progress.py

```diff
@@ -22,7 +22,9 @@ class OnboardingProgress(models.Model):
     company_id = fields.Many2one('res.company')
     onboarding_id = fields.Many2one(
         'onboarding.onboarding', 'Related onboarding tracked', required=True, ondelete='cascade')
-    progress_step_ids = fields.One2many('onboarding.progress.step', 'progress_id', 'Progress Steps Trackers')
+    progress_step_ids = fields.Many2many(
+        'onboarding.progress.step', string='Progress Steps Trackers', compute='_compute_progress_step_ids',
```

Comment:
Would add a clear example in commit message, to help understanding what we are modeling in this commit.